demo/src/plugins/dashboard/screens/dashboard/index.jsx

```diff
@@ -3,37 +3,74 @@ import PropTypes from 'prop-types';
 import NodeStatus from '@gdbots/schemas/gdbots/ncr/enums/NodeStatus';
 import { Col, Container, Row, Screen } from '@triniti/admin-ui-plugin/components';
 import TopArticles from '../../components/top-articles';
+import ActiveEdits from '../../components/active-edits';
+
+const Dashboard = ({ location, match: { params: { tab } } }) => {
+  const tabs = [
```

Comment:
@nanetteCodes do you need this? think you can just put the array straight into the prop

Reply:
updated